packages/web/src/pages/trending-page/components/RewardsBanner.tsx

```diff
@@ -121,17 +112,9 @@ const RewardsBanner = ({ bannerType }: RewardsBannerProps) => {
         </Text>
       </Flex>
       {!isMobile && (
-        <Flex
-          css={{
-            marginLeft: 'auto'
-          }}
-          alignItems='center'
-        >
-          <Text variant='label' size='l' strength='strong'>
-            {messages.learnMore}
-          </Text>
-          <IconCaretRight size='s' css={{ marginLeft: spacing.xs }} />
-        </Flex>
+        <PlainButton variant='inverted' size='large' iconRight={IconCaretRight}>
```

Comment:
ah since this doesn't have it's own click event, i think just make it "disabled" otherwise it looks a bit weird with the banner expanding and the button expanding